cas-server-integration-mongo/src/main/java/org/jasig/cas/services/convert/Converters.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.jasig.cas.services.convert;
+
+import com.google.common.cache.CacheLoader;
+import com.mongodb.DBObject;
+import org.apache.commons.logging.Log;
+import org.jasig.services.persondir.IPersonAttributes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.mongodb.core.convert.MappingMongoConverter;
+
+import javax.cache.Cache;
+import javax.cache.CacheManager;
+import javax.cache.configuration.MutableConfiguration;
+import javax.cache.integration.CacheWriter;
+import java.lang.ref.ReferenceQueue;
+import java.security.cert.CertPath;
+
+/**
+ * Collection of mongo converters that map objects to
+ * DB objects. Able to exclude types.
+ * @author Misagh Moayyed
+ * @since 4.1
+ */
+public final class Converters {
+    private Converters() {}
+
+    public static class NullConverter<I, O> implements Converter<I, O> {
+        protected final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+        @Override
+        public O convert(final I i) {
+            return null;
+        }
+    }
+
+    public static class LoggerConverter extends NullConverter<Logger, DBObject> {
```

Comment:
I trust you, but do we really have all these kinds of objects in services' definitions?